packages/react-textarea/src/stories/TextAreaBestPractices.md

```diff
@@ -0,0 +1,5 @@
+## Best practices
+
+### Do
+
+### Don't
```

Comment:
These should either contain content

Reply:
Yeah I'd say just delete the file for now and add it back later if someone gives you content for it

Reply:
I was about to push a change removing it :)